vizro-core/tests/unit/vizro/models/test_dashboard.py

```diff
@@ -149,27 +66,90 @@ def test_field_invalid_theme_input_type(self, page_1):
 class TestDashboardPreBuild:
     """Tests dashboard pre_build method."""
 
-    def test_dashboard_page_registry(self, prebuilt_two_page_dashboard, mock_page_registry):
-        result = dash.page_registry
-        expected = mock_page_registry
-        # Str conversion required as comparison of OrderedDict values result in False otherwise
-        assert str(result.items()) == str(expected.items())
```

Comment:
This was not a good way of doing the comparison before: actually comparing two dictionaries works fine and does not need `str`. The actual problem was the comparison of the `layout` functions, which is very tricky to get right, hence why I ignore them here.

Reply:
Yeees, the layout was the problem. Even better if we are fine ignoring it 👍